src/main/java/mods/railcraft/common/util/inventory/InvTools.java

```diff
@@ -602,23 +552,23 @@ public static boolean isItemEqual(ItemStack a, ItemStack b) {
      * @return True if equal
      */
     public static boolean isItemEqualIgnoreNBT(ItemStack a, ItemStack b) {
-        return isItemEqual(a, b, true, false);
+        return isItemEqual(a, b, true, false, false);
     }
 
-    public static boolean isItemEqual(final ItemStack a, final ItemStack b, final boolean matchDamage,
-            final boolean matchNBT) {
+    public static boolean isItemEqual(final ItemStack a, final ItemStack b, boolean matchSubtypes,
+            final boolean matchNBT, boolean matchDamage) {
```

Comment:
It's better to create *new* overloaded method and call it in existing one than changing arguments of existing method and patching all usages. I. e.:
```suggestion
    public static boolean isItemEqual(final ItemStack a, final ItemStack b, final boolean matchSubtypes,
            final boolean matchNBT) {
        return isItemEqual(a, b, matchSubtypes, matchNBT, false);
    }
    public static boolean isItemEqual(final ItemStack a, final ItemStack b, final boolean matchSubtypes,
            final boolean matchNBT, final boolean matchDamage) {
```

Reply:
@TimeConqueror I'm not talking about splitting logic into multiple small methods. I'm talking about backwards compatibility and ease of use. With the current implementation, you ***must*** change all usages of that method. With my approach (keep existing method and just call new one in its definition): if you *don't need* to change logic, then you *don't need* to change code (method usages). So instead of replacing the old method, you can just *extend* it.

Reply:
Similar change mitch did in CCL for the same reason. Backwards compatibilities (and not needing to update mods )
https://github.com/GTNewHorizons/CodeChickenLib/pull/16/files

Reply:
+1 to not breaking the public API